### PR TITLE
Ensure replacement color is positive

### PR DESCRIPTION
**Pull request by andreban**

### CODE REVIEW

**Review by nohe427 — APPROVED** [review]

File: packages/core/src/lib/ImageHelper.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export class ImageHelper {
     // See https://github.com/GoogleChromeLabs/bubblewrap/issues/488#issuecomment-806560923.
     if (backgroundColor && image.hasAlpha()) {
       // The replacement colour is the same as the background colour, but fully transparent.
-      const replacementColor = (backgroundColor.rgbNumber() << 8) & 0xFFFFFF00;
+      const replacementColor = ((backgroundColor.rgbNumber() << 8) & 0xFFFFFF00) >>> 0;
```
Comment (PEConn) [inline packages/core/src/lib/ImageHelper.ts:50]: This reads like magic to me (combined with there being no commit message). I'm assuming it's something to do with promoting the integer type?

Comment (andreban) [reply packages/core/src/lib/ImageHelper.ts:50]: TL;DR - bitten by the lack of unsigned integers :D. 

- `0xFFFFFF00` is `4294967040`, right?
- However, if you do `0xFFFFFF << 8`, you get `-256`.
- So, we force it to be interpreted as a positive int with `(0xFFFFFF << 8) >>> 0`

Comment (PEConn) [reply packages/core/src/lib/ImageHelper.ts:50]: What is the data type before and after the `>>> 0`?

Comment (andreban) [reply packages/core/src/lib/ImageHelper.ts:50]: `number`...

Comment (andreban) [reply packages/core/src/lib/ImageHelper.ts:50]: Adding more detailed comments on the next PR

**Pull request merged**